examples/motoko_examples/basic-dao/src/types.ts

```diff
@@ -0,0 +1,87 @@
+import { nat, Principal, Variant } from 'azle';
+
+export type Account = { owner: Principal; tokens: Tokens };
+export type Proposal = {
+  id: nat;
+  votes_no: Tokens;
+  voters: Principal[];
+  state: ProposalState;
+  timestamp: nat;
+  proposer: Principal;
+  votes_yes: Tokens;
+  payload: ProposalPayload;
+};
+export type ProposalPayload = {
+  method: string;
+  canister_id: Principal;
+  message: string;
+};
+
+export type ProposalState = Variant<{
+  // A failure occurred while executing the proposal
+  Failed?: string;
```

Comment:
If Motoko has all of these starting lowercase then I think we should do the same in TypeScript.